weave/__init__.py

```diff
@@ -44,4 +45,5 @@
     Evaluation,
     Scorer,
     AnnotationSpec,
+    SavedView,
```

Comment:
We have called this `SavedViews` internally as the project name. Curious if you think that is the best end-user name. How might we support savings non-call-table views in the future? Do you see a conflict here? Not sure how much `Saved` adds to this? Perhaps `CallView`? Probably not the biggest deal, but it will be something we need to maintain moving forward

Reply:
I think it is reasonable to expect we'd want to extend this capability for other tables (e.g. datasets, prompts) at some point, so I'd avoid using CallView.

I'm inclined not to just call it "View". That will make it harder to e.g. find this specific code amongst things like the term "viewer" or database-level views.

I'm open to other possibilities (TableView?, SharedView?) but none have come to mind that I think are strictly better, so I'm inclined to keep the name SavedView.